docs/api/regressor/influence.rst

```diff
@@ -0,0 +1,36 @@
+.. -*- mode: rst -*-
+
+Cook's Distance
+===============
+
+Cook's Distance is a measure of an observation or instances' influence on a linear
+regression. Instances with a large influence may be outliers and datasets that have a
+large number of highly influential points might not be good predictors to fit linear
+models. The ``CooksDistance`` visualizer shows a stem plot of all instances by index
+and their associated distance score, along with a heuristic threshold to quickly show
+what percent of the dataset may be impacting OLS regression models.
```

Comment:
I've added some basic documentation so that this is a documented visualizer, but I really would like to extend it. @pdamodaran @Kautumn06 should I do that in this PR or in a separate PR?